theme-settings.php

```diff
@@ -86,10 +86,22 @@ function galactus_form_system_theme_settings_alter(&$form, FormStateInterface &$
   $form['general']['clf_override'] = [
     '#type' => 'checkbox',
     '#title' => t('<strong>Use Minimal CLF</strong>'),
-    '#description' => t('Use an external, minimal copy of the CLS css files.<br><strong>Warning</strong>: Advanced users only. This will require creating and adding your own css for non-clf regions of the site, including navigation.'),
+    '#description' => t('Use an external, minimal copy of the CLS css files.<br><strong>Warning</strong>: Advanced users only. This will require creating and adding your own css for non-clf regions of the site, including navigation.<br><small><strong>Replaced by <i>CLF Version and Source</i> setting, kept for legacy purposes.</strong></small>'),
     '#default_value' => theme_get_setting('clf_override'),
   ];
 
+  $form['general']['clf_source'] = [
```

Comment:
Explanation: add new theme setting to define the source of the CLF (CDN full package, CDN minimal package, Local)